repository/p.json

```diff
@@ -1749,6 +1749,17 @@
 				}
 			]
 		},
+		{
+			"name": "Pluto Syntax Highlighting",
```

Comment:
You can simply name the package after the language itself, "Pluto" and leave out the "syntax highlighting" because it does quite a bit more than highlighting.